src/config/mongoose_config_parser_toml.erl

```diff
@@ -566,6 +615,26 @@ tls_cipher(#{<<"key_exchange">> := KEx,
     #{key_exchange => b2a(KEx), cipher => b2a(Cipher), mac => b2a(MAC), prf => b2a(PRF)};
 tls_cipher(Cipher) -> b2l(Cipher).
 
+set_overrides(State) ->
+    lists:foldl(fun(F, StateIn) -> F(StateIn) end, State, [
+        fun(S) -> maybe_override(S, override_global) end,
+        fun(S) -> maybe_override(S, override_local) end,
+        fun(S) -> maybe_override(S, override_acls) end
+        ]).
+
+maybe_override(S = {state, Config, _, _, _, _}, Name) ->
+    case lists:keyfind(Name, 2, Config) of
+        {_, _, true} ->
+            S1 = mongoose_config_parser:Name(S),
+            C1 = lists:keydelete(Name, 2, Config),
+            mongoose_config_parser:set_opts(C1, S1);
+        {_, _, false} ->
```

Comment:
You are right that `keyfind` returns a tuple or `false` atom. That is exactly why it is done this way, because if there is a `false` value in the config file i.e.
```
override_local = false
```
Then we will match on a tuple and we want to remove it from the config, without changing the `#state.override_local` value. If we don't match, there is no such tuple and no need for any action - that's why there is this distinction in the cases.